packages/types/CHANGELOG.json

```diff
@@ -2,10 +2,6 @@
     {
         "version": "2.5.0-beta.1",
         "changes": [
-            {
```

Comment:
Was this already published with `SendTransactionOpts`? If so it might need a version bump with a revert Changelog.

Reply:
Good catch, removed this change and added a revert changelog entry.